docs/userguide/containers/dockerizing.md

```diff
@@ -24,147 +24,131 @@ Running an application inside a container takes a single command: `docker run`.
     /
     root@af8bae53bdd3:/# ls
     bin boot dev etc home lib lib64 media mnt opt proc root run sbin srv sys tmp usr var
 
-You can see we've run the `pwd` to show our current directory and can
-see we're in the `/` root directory. We've also done a directory listing
-of the root directory which shows us what looks like a typical Linux
-file system.
+In this example:
 
-You can play around inside this container and when you're done you can
-use the `exit` command or enter Ctrl-D to finish.
+* `pwd` displays the current directory, the `/` root directory.  
+* `ls` displays the directory listing of the root directory of a typical Linux file system.
 
-    root@af8bae53bdd3:/# exit
+Now, you can play around inside this container. When completed, run the `exit` command or enter Ctrl-D
+to exit the interactive shell.
 
-As with our previous container, once the Bash shell process has
-finished, the container is stopped.
+    root@af8bae53bdd3:/# exit
 
-## A daemonized Hello world
+>**Note:** As with our previous container, once the Bash shell process has
+finished, the container stops.
 
-Now a container that runs a command and then exits has some uses but
-it's not overly helpful. Let's create a container that runs as a daemon,
-like most of the applications we're probably going to run with Docker.
+## Start a daemonized Hello world
 
-Again we can do this with the `docker run` command:
+Let's create a container that runs as a daemon.
 
     $ docker run -d ubuntu /bin/sh -c "while true; do echo hello world; sleep 1; done"
     1e5535038e285177d5214659a068137486f96ee5c2e85a4ac52dc83f2ebe4147
 
-Wait, what? Where's our "hello world" output? Let's look at what we've run here.
-It should look pretty familiar. We ran `docker run` but this time we
-specified a flag: `-d`. The `-d` flag tells Docker to run the container
-and put it in the background, to daemonize it.
+In this example:
 
-We also specified the same image: `ubuntu`.
+* `docker run` runs the container.
+* `-d` flag runs the container in the background (to daemonize it).
+* `ubuntu` is the image you would like to run.
 
-Finally, we specified a command to run:
+Finally, we specify a command to run:
 
     /bin/sh -c "while true; do echo hello world; sleep 1; done"
 
-This is the (hello) world's silliest daemon: a shell script that echoes
-`hello world` forever.
 
-So why aren't we seeing any `hello world`'s? Instead Docker has returned
-a really long string:
+In the output, we do not see `hello world` but a long string:
 
     1e5535038e285177d5214659a068137486f96ee5c2e85a4ac52dc83f2ebe4147
 
-This really long string is called a *container ID*. It uniquely
+This long string is called a *container ID*. It uniquely
 identifies a container so we can work with it.
 
 > **Note:**
-> The container ID is a bit long and unwieldy. A bit later,
-> we'll see a shorter ID and ways to name our containers to make
+> The container ID is a bit long and unwieldy. Later, we will cover the short
```

Comment:
This page doesn't seem to use notes very well, my  guidance would be notes should be used to call out exceptions to the topic under discussion or relevant related information that might trip a user up.  Here they seem be arbitrarily used to expand on the context.  There isn't any reason not to do that in the body text.

![image](https://cloud.githubusercontent.com/assets/1347209/13750116/621585b6-e9c1-11e5-8e19-d18b5c19c0e5.png)